war/src/main/js/util/security.js

```diff
@@ -0,0 +1,6 @@
+function sanitizeString(str) {
+  str = str.replace(/[^a-z0-9áéíóúñü .,_-]/gim, "");
```

Comment:
It won't work, if we take the example of the Russian, Chinese, Japanese alphabet, etc., nothing will be displayed.

The best approach will be to convert dangerous characters into their HTML encoded version, you can find an example with the function [Util.xmlEscape](https://github.com/jenkinsci/jenkins/blob/d3d1c88d299735186209ad5b2f7a41e05486105f/core/src/main/java/hudson/Util.java#L1045).